aircraft/global5000/Systems/GNCUtilities.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0"?>
```

Comment:
Is this version of GNCUtilities.xml any different to the original version in jsbsim/systems? If not can it not be used/included from there?

Reply:
how to do it? just remove them all?